bus/bus.go

```diff
@@ -649,6 +655,59 @@ func (b *Bus) broadcastContract(ctx context.Context, fcid types.FileContractID)
 	return
 }
 
+func (b *Bus) fetchSetting(ctx context.Context, key string) (interface{}, error) {
```

Comment:
This could move to a settings manager 👀

Reply:
This reduces type safety and hides the implementation without significant benefit imo. Having individual methods like `fetchGougingSettingsOrDefault` would be my preference.